src/core/templates/show-people.js

```diff
@@ -114,7 +93,23 @@ export default function showPeople(persons = []) {
     }
     if (p.orcid) {
       contents.push(
-        html`<a class="p-name orcid" href="${p.orcid}">${orcidIcon} </a>`
+        html`<a class="p-name orcid" href="${p.orcid}"
```

Comment:
I think we want to do `orcidIcon.cloneNode(true)` instead, as the previous code was moving `orcidIcon` around the parents.

Reply:
Oh my! What would I do without you!